test/test_nn.py

```diff
@@ -9365,6 +9099,239 @@ def _test_module_empty_input(self, module, inp, check_size=True):
                 self.assertEqual(p.grad, torch.zeros_like(p.grad))
         self.assertEqual(inp.grad, torch.zeros_like(inp))
 
+    @unittest.skipIf((not TEST_NUMPY) or (not TEST_SCIPY) or (scipy.__version__ < '1.0.0'),
+                     "Scipy v1.0 and/or numpy not found")
+    @tf32_on_and_off()
+    def test_affine_2d_rotate0(self, device):
```

Comment:
@ngimel asks: do these functions actually use TF32? 

Moving this test is great, but can you elaborate on why you moved it?

Reply:
The following tests seem to suggest these functions ARE using TF32, since they are being given different precisions to pass. If they do use TF32, however, the documentation needs to be updated to reflect that the flag affects convolutions, lstms, and ...

We noticed, while reviewing, that code for convolutions and lstms had been updated to check the TF32 flag - how does the global flag affect these other functions?

Reply:
Documents updated to contain a list of affected things. The flags affect functions by cuBLAS handle and cuDNN algo. Other functions, if affected, are affected by calling cuBLAS bindings like `gemm` or calling cuDNN convolutions.